### PR TITLE
feat(appbar): fixed image size

### PR DESCRIPTION
**Pull request by Yekez**
change quality for filter purposes

### CODE REVIEW

**Comment by Yekez** [comment]
https://tercuman-bulmaca--pr25-main-7ifdoukt.web.app/assets/null/logo_horiz.png

image assets are not loaded properly. maybe deploy is ignoring public folder?

**Comment by Yekez** [comment]
Issue might be firebase related.

File: lib/resources/widgets/molecules/main_app_bar.dart [hunk 1]
```diff
@@ -14,13 +14,14 @@ class MainAppBar extends StatelessWidget implements PreferredSizeWidget {
...
+            constraints: BoxConstraints(
+              maxWidth: 225,
+              maxHeight: 28.54,
+            ),
             child: Padding(
               padding: const EdgeInsets.symmetric(horizontal: 16.0),
-              child: Image.asset(
-                getImageAsset("logo_horiz.png"),
-              ),
+              child: Image.asset("images/logo_lq.png"),
```
Comment (xsorifc28) [inline lib/resources/widgets/molecules/main_app_bar.dart:24]: Use getImageAsset (see old code)

Comment (xsorifc28) [reply lib/resources/widgets/molecules/main_app_bar.dart:24]: Can use the existing image, it should work also.

Comment (Yekez) [reply lib/resources/widgets/molecules/main_app_bar.dart:24]: existing high quality image looks sawtooth-y. I have tested filters and anti aliasing. Turns out, best looking image is the one I exported from figma.

Comment (xsorifc28) [reply lib/resources/widgets/molecules/main_app_bar.dart:24]: Ok, let's update the code and merge it. I think you just need `getImageAsset("logo_lq.png")` and only upload the image to `/public/...`

Also, I already merged a different PR with `ConstrainedBox` change.

**Comment by Yekez** [comment]
recent changes will make this pr obsolete. closing.

**Pull request closed without merging**